### PR TITLE
fix(curriculum): update test instruction wording

### LINKED ISSUE #55765: Build a Budget App Chart hint
### Describe the Issue

Problem:

You absolutely need the Assertion error to complete the chart but many people miss the small hint at the end of the instructions "Note: open the browser console with F12 to see a more verbose output of the tests.". It's right after the chart so I think it's easy to miss.

Proposals:

- Put the note in the fCC console: "// running tests
create_spend_chart should print a different chart representation. Check that all spacing is exact. Open the browser console with F12 to see a more verbose output of the tests.
// tests completed "

- Put the note in the Tests area: "create_spend_chart should print a different chart representation. Check that all spacing is exact. Open the browser console with F12 to see a more verbose output of the tests."

- Put the Assertion Error directly in the FCC console area

- Add more explicit instructions for spacing. This could be in the format: 'Percentage spent by category\n100| ' The Arithmetic Formatter has similar feedback.

- Could also be visually where spaces and newlines are indicated. You can actually highlight the chart in the instructions and it makes the spacing more apparent. Screenshot attached.

### Affected Page

https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-budget-app-project/build-a-budget-app-project

### Your code

N/A

### Expected behavior

N/A

### Screenshots

Screenshot 2024-03-10 211440

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by LouizaMak**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes [#55765](https://github.com/freeCodeCamp/freeCodeCamp/issues/55765)

Update original test instructions: 

> `create_spend_chart` should print a different chart representation. Check that all spacing is exact.

To: 

> `create_spend_chart` should print a different chart representation. Check that all spacing is exact. Open your browser console with F12 for more details.

### CODE REVIEW

**Review by gikf — APPROVED** [review]
Note for future - it is recommended to make changes on a separate branch, rather than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/how-to-contribute-to-the-codebase/#contributing-to-the-codebase.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-oop-by-building-a-shopping-cart/63ee5d8f9e7168076e932fe2.md
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-oop-by-building-a-shopping-cart/63ee5d8f9e7168076e932fe2.md:1]: This change is not related to the linked issue. Please remove this modification from the PR.

**Review by gikf — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-oop-by-building-a-shopping-cart/63ee5d8f9e7168076e932fe2.md
Comment (LouizaMak) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-oop-by-building-a-shopping-cart/63ee5d8f9e7168076e932fe2.md:1]: Apologies for the confusion, modification has been removed from the PR.

**Review by gikf — APPROVED** [review]

**Comment by sreevamsi2005** [comment]
yes its fine i have verified

**Review by huyenltnguyen — APPROVED** [review]
